code/script.js

```diff
@@ -1,9 +1,61 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const nameInput = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper')
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
+const handleNameInput = (event) => {
+event.preventDefault()
+
+  const name = nameInput.value
+  showMessage (name, 'user')
+  nameInput.value = ''
+
+  setTimeout(()=> showWebsiteOptions (name), 1000)
+}
+
+const showWebsiteOptions = (userName) => {
+  //Post a greeting to the user as the bot
+  showMessage (`Nice to meet you, ${userName}. My name is Dave, and I am here to help you navigate this 
+  website. Which pages would you be most interested in visiting?`, 'bot')
+//Update the input wrapper with the next set of options
+inputWrapper.innerHTML = `
+<select id="Dropdown">
+<option value="">So, what are you interested in? </option>
+<option value="Children's Decor">Children's Decor</option>
+<option value="Adult's Decor">Adult's Decor</option>
+<option value="Birthday Ideas">Birthday Ideas</option>
+<option value="Artwork">Artwork</option>
+</select>`
+}
+
+
+
+const dropdown = () => {
+const value = document.getElementById('Dropdown').value
+showMessage(value, 'user')
+
+
+setTimeout(()=> botReply (value), 1000)
+inputWrapper.innerHTML=''
+}
+const botReply = (userValue) =>{
+showMessage (`${userValue}? That's a great choice`, 'bot')
+if (userValue === "Children's Decor") {  
+showMessage (`If you are interested in ${userValue}, 
+then I suggest you look at <a href = './isa-room.html'> this page</a>`, 'bot')} else if (userValue ==="Adult's Decor")
+{showMessage (`If you are interested in ${userValue}, 
+then I suggest you look at <a href = './playroom.html'> this page</a>`, 'bot')} else if (userValue ==="Birthday Ideas")
+{showMessage (`If you are interested in ${userValue}, 
+then I suggest you look at <a href = './menu.html'> this page</a>`, 'bot')} else if (userValue ==="Artwork")
+{showMessage (`If you are interested in ${userValue}, 
+then I suggest you look at <a href = './diamond.html'> this page</a>`, 'bot')}
+
+}
```

Comment:
This block of code is a little hard to read. If you indent correctly it will be much easier to read and also to debug (e.g. see if a curly bracket is missing). Also when naming functions, it's good to have a verb like "show" or similar so that you explain what the function does.

```
const handleDropdownSelection = () => {
  const value = document.getElementById('Dropdown').value
  showMessage(value, 'user')

  setTimeout(()=> botReply (value), 1000)
  inputWrapper.innerHTML=''
}
```

```
const botReply = (userValue) =>{
  showMessage (`${userValue}? That's a great choice`, 'bot')
  if (userValue === "Children's Decor") {  
    showMessage (`If you are interested in ${userValue}, 
then I suggest you look at <a href = './isa-room.html'> this page</a>`, 'bot')
  } else if (userValue ==="Adult's Decor") {
    showMessage (`If you are interested in ${userValue}, 
then I suggest you look at <a href = './playroom.html'> this page</a>`, 'bot')
  } else if (userValue ==="Birthday Ideas") {
    showMessage (`If you are interested in ${userValue}, 
then I suggest you look at <a href = './menu.html'> this page</a>`, 'bot')
  } else if (userValue ==="Artwork") {
    showMessage (`If you are interested in ${userValue}, 
then I suggest you look at <a href = './diamond.html'> this page</a>`, 'bot')
  }
}
```